bicep/analysis.py

```diff
@@ -121,16 +218,139 @@ def plot_panel_capacity(self, residential=1, log_y=True):
+        # after all states have completed, combine results 
+        # list for aggregate cost data results
+        aggregate_data = []
+        # list for buildings df data
+        buildings_list = []
+        # list for meta df data
+        meta_list = []
 
-    print(f'total commercial cost for bau: ${bau.total_commercial_costs:,.0f}')
-    print(f'total commercial cost for high: ${high.total_commercial_costs:,.0f}')
+        for state in self.all_states:
+            if state in final_results:
+                # combined aggregate data 
+                aggregate_data.append({
+                    'state': state,
+                    'total_cost': final_results[state].total_cost,
+                    'total_residential_costs': final_results[state].total_residential_costs,
+                    'total_commercial_costs': final_results[state].total_commercial_costs
+                })
 
+                # collect DataFrames for concatenation 
+                buildings_list.append(final_results[state].buildings)
+                meta_list.append(final_results[state].building_meta)
+
+        # Create final DataFrames
+        self.all_states_results = pd.DataFrame(aggregate_data)
+        
+        # Filter out empty DataFrames to avoid FutureWarning
+        non_empty_buildings = [df for df in buildings_list if not df.empty]
+        non_empty_meta = [df for df in meta_list if not df.empty]
+        
+        self.all_states_buildings = pd.concat(non_empty_buildings, ignore_index=True) if non_empty_buildings else pd.DataFrame()
+        self.all_states_meta = pd.concat(non_empty_meta, ignore_index=True) if non_empty_meta else pd.DataFrame()
+        
+        # Calculate and display national totals
+        self.total_cost = self.all_states_results['total_cost'].sum()
+        self.total_residential_costs = self.all_states_results['total_residential_costs'].sum()
+        self.total_commercial_costs = self.all_states_results['total_commercial_costs'].sum()
+        
+        logger.info(f"=== FINAL NATIONAL TOTALS (Individual State Method) ===")
+        logger.info(f"Total Cost: ${self.total_cost:,.0f}")
+        logger.info(f"Residential Cost: ${self.total_residential_costs:,.0f}")
+        logger.info(f"Commercial Cost: ${self.total_commercial_costs:,.0f}")
+        
+        # Save results if requested
+        if self.save_results:
+            self._save_individual_state_results()
+        
+        logger.info("=== Individual State Analysis Complete ===")
+    
+    def _save_individual_state_results(self):
+        """Save results from individual state analysis with standard naming convention"""
+        from utils.config import PARSED_INPUTS_PATH
+        
+        PARSED_INPUTS_PATH.mkdir(parents=True, exist_ok=True)
+        
+        # Save aggregate cost summary results
+        self.all_states_results.to_csv(f"data/parsed_inputs/bicep_cost_summary_{self.scenario}.csv", index=False)
```

Comment:
You should use the pathlib path when saving instead of hard coding that path. 

```suggestion
        self.all_states_results.to_csv(PARSED_INPUTS_PATH / f"bicep_cost_summary_{self.scenario}.csv", index=False)
```